modules/xfem/test/tests/moving_interface/moving_ad_bimaterial_finite_strain.i

```diff
@@ -0,0 +1,271 @@
+# This test is for two layer materials with different youngs modulus with AD
+# The global stress is determined by switching the stress based on level set values
+# The material interface is marked by a level set function
+# The two layer materials are glued together
+# This case is also meant to test for a bug in moving interfaces on displaced meshes
```

Comment:
Since I assume the bug doesn't exist any more, I would reword this to not mention the bug, but say what the scenario that it tests to make sure works is.